runtime/compiler/runtime/MetaData.cpp

```diff
@@ -1469,8 +1465,6 @@ createMethodMetaData(
          sizeOfStackAtlasInBytes,
          nonmergeableBCI);
 
-   data->registerSaveDescription = comp->cg()->getRegisterSaveDescription();
-
```

Comment:
This is still needed.  This is a different "registerSaveDescription" that's used to describe registers preserved in the prologue.

Reply:
This may very well be the cause of your CI failures.